### ISSUE TITLE
Some bugs in version@1.7.8

### ISSUE THREAD

**Issue by kairyou** [hung-phan/koa-react-isomorphic#28]
Hi, 
-  `del` and `debug` are missing in package.json
```shell
# npm run watch
module.js:327
    throw err;
    ^
Error: Cannot find module 'del'
# ...
ERROR in ./app/server-app.js
Module not found: Error: Can't resolve 'debug' in '../koa-react-isomorphic/app'
 @ ./app/server-app.js 11:13-29
```
- bugs with multi router in `relay` branch (`SERVER_RENDERING=true npm run watch` && `npm run dev`)
  + `http://localhost:3000/test` 

     The content of `div#app` is replaced with empty.
     ![image](https://cloud.githubusercontent.com/assets/544524/16144083/705354dc-34a3-11e6-826b-9ed34f0a3e5b.png)
  + `http://localhost:3000/` 

     The URL in the address bar is not changed after clicked the `/test` link on the home page.

```js
// app/routes.js (add route: /test)
import Test from 'client/components/test';
//...
<Route path="/" component={Todos} queries={ViewerQuery} />
<Route path="/test" component={Test} />
```

```js
// components/test/index.js (add component: test)
import React from 'react';
import { Link } from 'react-router';
const Test = () => (
  <div className="container">
      <Link to="/">Index</Link>
  </div>
);
export default Test;
```

```js
// components/todos/index.js (add link: /test)
import { Link } from 'react-router';
// ...
<TodosBody viewer={viewer} />
<Link to="/test">test</Link>
```

Suggest add `/test` route by default.
Thanks :)

**Comment by hung-phan** [comment]
Ok :))

**Comment by hung-phan** [comment]
https://github.com/denvned/isomorphic-relay-router/issues/37

**Comment by hung-phan** [comment]
Resolved with 1.7.9 @kairyou

**Comment by kairyou** [comment]
Thanks 👍
